gcc-8.2 view::persist compile error

Building target `make view_persist_test` produces:

```terminal
In file included from /seqan3/submodules/range-v3/include/range/v3/begin_end.hpp:27,
                 from /seqan3/submodules/range-v3/include/range/v3/algorithm/copy.hpp:19,
                 from /seqan3/test/unit/range/view/view_persist_test.cpp:39:
/seqan3/submodules/range-v3/include/range/v3/utility/iterator.hpp: In instantiation of ‘ranges::v3::back_insert_iterator<Container>& ranges::v3::back_insert_iterator<Container>::operator=(const typename Container::value_type&) [with Container = std::basic_string_view<char>; typename Container::value_type = char]’:
/seqan3/submodules/range-v3/include/range/v3/algorithm/copy.hpp:50:26:   required from ‘constexpr ranges::v3::tagged_pair<ranges::v3::tag::in(I), ranges::v3::tag::out(O)> ranges::v3::copy_fn::operator()(I, S, O) const [with I = __gnu_cxx::__normal_iterator<char*, std::__cxx11::basic_string<char> >; S = __gnu_cxx::__normal_iterator<char*, std::__cxx11::basic_string<char> >; O = ranges::v3::back_insert_iterator<std::basic_string_view<char> >; int _concept_requires_44 = 42; typename std::enable_if<((_concept_requires_44 == 43) || (((ranges::v3::concepts::models<ranges::v3::concepts::InputIterator, I>() && ranges::v3::concepts::models<ranges::v3::concepts::Sentinel, S, I>()) && ranges::v3::concepts::models<ranges::v3::concepts::WeaklyIncrementable, Out>()) && ranges::v3::concepts::models<ranges::v3::concepts::IndirectlyCopyable, I0, Out>())), int>::type <anonymous> = 0; ranges::v3::tagged_pair<ranges::v3::tag::in(I), ranges::v3::tag::out(O)> = ranges::v3::tagged_detail::tagged<std::pair<__gnu_cxx::__normal_iterator<char*, std::__cxx11::basic_string<char> >, ranges::v3::back_insert_iterator<std::basic_string_view<char> > >, ranges::v3::tag::in, ranges::v3::tag::out>; typename meta::v1::detail::front_<typename meta::v1::detail::as_list_<ranges::v3::tag::in(I)>::type>::type = ranges::v3::tag::in; typename meta::v1::detail::front_<typename meta::v1::detail::as_list_<ranges::v3::tag::out(O)>::type>::type = ranges::v3::tag::out; typename meta::v1::detail::back_<typename meta::v1::detail::as_list_<ranges::v3::tag::out(O)>::type>::type = ranges::v3::back_insert_iterator<std::basic_string_view<char> >; typename meta::v1::detail::back_<typename meta::v1::detail::as_list_<ranges::v3::tag::in(I)>::type>::type = __gnu_cxx::__normal_iterator<char*, std::__cxx11::basic_string<char> >]’
/seqan3/include/seqan3/range/view/persist.hpp:210:21:   required from ‘seqan3::detail::view_persist<urng_t>::operator container_t() const requires  CommonReference<typename seqan3::reference<container_t>::type, typename seqan3::reference::type> and const_iterable_concept<urng_t> [with container_t = std::basic_string_view<char>; urng_t = std::__cxx11::basic_string<char>]’
/usr/include/c++/8.2.1/bits/basic_string.h:640:47:   required from ‘std::__cxx11::basic_string<_CharT, _Traits, _Alloc>::basic_string(const _Tp&, const _Alloc&) [with _Tp = seqan3::detail::view_persist<std::__cxx11::basic_string<char> >; <template-parameter-2-2> = void; _CharT = char; _Traits = std::char_traits<char>; _Alloc = std::allocator<char>]’
/seqan3/test/unit/range/view/view_persist_test.cpp:77:5:   required from here
/seqan3/submodules/range-v3/include/range/v3/utility/iterator.hpp:415:29: error: ‘class std::basic_string_view<char>’ has no member named ‘push_back’
                 container_->push_back(value);
                 ~~~~~~~~~~~~^~~~~~~~~
```

--- a/include/seqan3/range/view/persist.hpp
+++ b/include/seqan3/range/view/persist.hpp
@@ -183,11 +183,11 @@ public:
     //!\}
 
     /*!\brief Convert this view into a container implicitly.
-     * \tparam container_t Type of the container to convert to; must satisfy seqan3::container_concept and the
+     * \tparam container_t Type of the container to convert to; must satisfy seqan3::sequence_container_concept and the
      *                     seqan3::reference_t of both must model std::CommonReference.
      * \returns This view converted to container_t.
      */
-    template <container_concept container_t>
+    template <sequence_container_concept container_t>
     operator container_t()
     //!\cond
         requires std::CommonReference<reference_t<container_t>, reference>
@@ -199,7 +199,7 @@ public:
     }
 
     //!\overload
-    template <container_concept container_t>
+    template <sequence_container_concept container_t>
     operator container_t() const
     //!\cond
         requires std::CommonReference<reference_t<container_t>, reference> &&

